PKGBUILD

```diff
@@ -188,15 +184,14 @@ ac_add_options --with-mozilla-api-keyfile=${PWD@Q}/mozilla-api-key
 
 # System Libraries
 ac_add_options --with-system-nspr
-ac_add_options --with-system-nss
+#ac_add_options --with-system-nss
```

Comment:
Just making a note that this is not the Arch-way. Some apps like Discord don't like linking against a differnt NSS than Firefox. But if this is a good fix, then one can work on re-activating nss later.